File: mw/lvech.m
function [v] =  lvech(S)
% Computes lower triangle (below main diagonal) and puts in vector v
n = size(S,1);
v = NaN;
for i = 1:n-1;
    v = [v;S(i+1:end,i)];
end;
v = v(2:end,1);

end

File: R/lvech.R
#' Extract Lower Triangle (Excluding Diagonal)
#'
#' Extracts the elements of a matrix that are strictly below the main diagonal
#' (excluding the diagonal) and returns them as a vector.
#'
#' @param S A numeric matrix.
#' @return A numeric vector containing the elements below the main diagonal.
lvech <- function(S) {
  # Extract elements strictly below the main diagonal (diag = FALSE)
  return(S[lower.tri(S, diag = FALSE)])
}